<launch>
    <remap from="lstate" to="/left_wheel/state"/>
    <remap from="rstate" to="/right_wheel/state"/>
    <remap from="lcontrol_effort" to="/left_wheel/control_effort"/>
    <remap from="rcontrol_effort" to="/right_wheel/control_effort"/>
    <node name="left_wheel_pid" pkg="pid" type="controller" ns="left_wheel">
          <param name="Kp" value="0.4" />
          <param name="Ki" value="0.14" />
          <param name="Kd" value="0.0" />
    </node>
    <node name="right_wheel_pid" pkg="pid" type="controller" ns="right_wheel">
          <param name="Kp" value="0.4" />
          <param name="Ki" value="0.14" />
          <param name="Kd" value="0.0" />
    </node>
    <node
        name="motor_control"
        pkg="club_bot3"
        type="DiffDrive_Motor_node.py"
        respawn="true"
        launch-prefix="xterm -e"
    />
    <node
        pkg="rosserial_python"
        type="serial_node.py"
        name="arduino_node"
        >
            <param
                name="port"
                value="/dev/ttyACM0"
            />
    </node>
</launch>
